### ISSUE TITLE
Langflow not loading all required Environment variable mentioned in LANGFLOW_VARIABLES_TO_GET_FROM_ENVIRONMENT

### ISSUE THREAD

**Issue by codenprogressive** [langflow-ai/langflow#2520]
**Describe the bug**
Trying to load 3 global variables from Environment. only the first variable is loaded and not the second

**Browser and Version**
 - Browser: Edge
 - Version: 126.0.2592.81

**To Reproduce**
1. my .env file: includes the lines:

```python

LANGFLOW_STORE_ENVIRONMENT_VARIABLES = true
LANGFLOW_VARIABLES_TO_GET_FROM_ENVIRONMENT='["default_gcp_project", "default_gcp_location", "default_gcp_dataset"]'
```
2. launch Langflow: `python -m langflow run  --components-path /src_backend_platform/ --env-file /env/.env`
3. Only the first variable is loaded in Langflow: `default_gcp_project`

**Screenshots**

![image](https://github.com/langflow-ai/langflow/assets/36305975/cd106870-c6e1-4e92-bb20-be1394c14621)

**Comment by liwalayouni** [comment]
@dosu, I made sure the .env is as the following:

```
LANGFLOW_STORE_ENVIRONMENT_VARIABLES=true
LANGFLOW_VARIABLES_TO_GET_FROM_ENVIRONMENT='["default_gcp_location", "default_gcp_dataset", "default_gcp_project"]'
```

this is still resulting into loading only the first variable.

**Comment by liwalayouni** [comment]
Unfortunately still not working! looks like it is a bug. Anyone else can confirm able to load multiple environment variable?

**Comment by peteBarahon** [comment]
I just posted a bug #2717 a few hours ago which may relate to your issue.

**Comment by italojohnny** [comment]
Hi, @codenprogressive . How are you?

The issue has been fixed in PR #2971. I tested it and was able to load the environment variables you mentioned, as shown in the image:

![20240729070019_624x637](https://github.com/user-attachments/assets/e2de2e03-5961-4554-bfb3-443a68739a15)


If you encounter any other issues, please open a new issue!

Thank you very much for your collaboration!
